ui/apps/platform/src/Containers/Clusters/DelegateScanning/DelegateScanningPage.tsx

```diff
@@ -1,21 +1,64 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import {
+    Alert,
+    AlertVariant,
     Breadcrumb,
     BreadcrumbItem,
     Card,
     CardBody,
     Flex,
     FlexItem,
     PageSection,
+    Radio,
     Title,
 } from '@patternfly/react-core';
 
 import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
 import PageTitle from 'Components/PageTitle';
 import { clustersBasePath } from 'routePaths';
+import { fetchDelegatedRegistryConfig } from 'services/DelegatedRegistryConfigService';
+import { DelegatedRegistryConfig } from 'types/dedicatedRegistryConfig.proto';
+import ToggleDelegatedScanning from './Components/ToggleDelegatedScanning';
+
+const initialDelegatedState: DelegatedRegistryConfig = {
+    enabledFor: 'NONE',
+    defaultClusterId: '',
+    registries: [],
+};
 
 function DelegateScanningPage() {
     const displayedPageTitle = 'Delegate Image Scanning';
+    const [delegatedRegistryConfig, setDedicatedRegistryConfig] =
+        useState<DelegatedRegistryConfig>(initialDelegatedState);
+    const [errMessage, setErrMessage] = useState<string | null>(null);
+
+    useEffect(() => {
+        fetchDelegatedRegistryConfig()
+            .then((result) => {
+                setDedicatedRegistryConfig(result.response);
```

Comment:
Many components clear the error message on success.

Which brings up a question about pro and con of `null` versus empty string `''` for no message state:
* `!!errMessage` below is compatible with either
* A components that are concerned that actual error message might be empty use `|| 'whatever` idiom, so a non-string falsey value is not necessary to cover all of the cases.

Reply:
`getAxiosErrorMessage` util function guarantees an error string, at least "Unknown error" in the last desperate extreme.

Reply:
As it's currently coded, this is API call is only made on component load, because any changes will be made being PUT from this page, so it should not need to be refetched.

We may decide that we do need a way to refetch after an error, in which case we can move it out to its own function, and just call it from the onload `useEffect`.